CHANGELOG.md

```diff
@@ -13,7 +26,7 @@ All notable changes to this project will be documented in this file.
 
 - Jax template now uses equinox `filter_jit` to allow non-array inputs (#56)
 - Added pip as dependency (#58)
-- Buggy `is_leaf` check in the `abstract_eval` of jax recipe and vectoradd_jax (#75)
```

Comment:
dang, old stuff gets auto-renamed 🙄

Reply:
I suggest we live with this, but be more strict with pr names going forward. I don't think interactive rebase and changing the name of that commit + force-push to main is a good idea. @dionhaefner @apaleyes for awareness

(btw, that PR title was mine 🙈 )

Reply:
agreed